code/modules/crafting/recipes.dm

```diff
@@ -177,4 +177,19 @@
 				/obj/item/weaponcrafting/stock = 1,
 				/obj/item/stack/packageWrap = 5,)
 	tools = list(/obj/item/weapon/screwdriver)
-	time = 200
\ No newline at end of file
+	time = 200
+
+/datum/table_recipe/spooky_camera
+	name = "Camera Obscura"
+	result = /obj/item/device/camera/spooky
+	time = 15
+	reqs = list(/obj/item/device/camera = 1,
+				/datum/reagent/holywater = 10)
```

Comment:
I'm not sure if this actually works, we never upgraded our tablecrafting to -tg-'s.